setup.cfg

```diff
@@ -0,0 +1,43 @@
+[metadata]
+name = enetpy
+author = Thermondo GmbH
+author-email = opensource@thermondo.com
+summary = Python wrapper for the ene't Marktpartner API
+long_description = file: README.md
+long_description_content_type = text/markdown; charset=UTF-8
+home-page = https://github.com/Thermondo/python-enet
+project_urls =
+    Bug Tracker = https://github.com/Thermondo/python-enet/issues
+    Documentation = https://github.com/Thermondo/python-enet
+    Source Code = https://github.com/Thermondo/python-enet
+license = MIT License
+classifier =
+    Development Status :: 5 - Production/Stable
```

Comment:
Is it already production and stable? :)

Reply:
The python API will be stable, won't it?